lib/pgBackRest/Config/Config.pm

```diff
@@ -2216,32 +194,33 @@ sub configLoad
     # If nothing was passed on the command line then display help
     if (@ARGV == 0)
     {
-        commandSet(CMD_HELP);
+        cfgCommandSet(CFGCMD_HELP);
     }
     # Else process command line options
     else
     {
         # Parse command line options
-        if (!GetOptions(\%oOptionTest, %oOptionAllow))
+        if (!GetOptions(\%oOptionTest, @stryOptionAllow))
         {
-            commandSet(CMD_HELP);
+            cfgCommandName(CFGCMD_HELP);
             return false;
         }
 
         # Validate and store options
         my $bHelp = false;
 
-        if (defined($ARGV[0]) && $ARGV[0] eq CMD_HELP && defined($ARGV[1]))
+        if (defined($ARGV[0]) && $ARGV[0] eq cfgCommandName(CFGCMD_HELP) && defined($ARGV[1]))
```

Comment:
Hmmm, when I built the docs and then entered the doc-db-master, I typed sudo -u postgres pgbackrest help check
and got:
Segmentation fault (core dumped)
but
 sudo -u postgres pgbackrest help info
 does work.